src/System.Management.Automation/engine/CommandDiscovery.cs

```diff
@@ -1286,27 +1286,6 @@ internal void UnregisterLookupCommandInfoAction(string currentAction, string com
         private HashSet<string> _activeCommandNotFound = new HashSet<string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Online help tests for PowerShell Cmdlets.Validate 'get-help Stop-Computer -Online'
```powershell
Expected strings to be the same, but they were different.
Expected length: 47
Actual length:   46
Strings differ at index 40.
Expected: '...nkid=20971...'
But was:  '...nkID=13526...'
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Help/HelpSystem.OnlineHelp.Tests.ps1: line 48
48:                 $actualURI | Should -Be $cmdlet.HelpURI
```